content/news/special-announcement.md

```diff
@@ -1,3 +1,8 @@
+<p><i>Update 2018-14-10:</i></p>
```

Comment:
Is that the correct date format (I really don't know)?

Reply:
Oops, fixed.